src/components/Select/styled/error.js

```diff
@@ -1,13 +1,16 @@
 import styled from 'styled-components';
 import { FONT_SIZE_TEXT_MEDIUM } from '../../../styles/fontSizes';
 import { MARGIN_X_SMALL } from '../../../styles/margins';
-import { COLOR_ERROR } from '../../../styles/colors';
+import getTheme from '../../../styles/helpers/getTheme';
 
-const StyledError = styled.div`
+const StyledError = styled.div.attrs(props => {
+    const errorMainColor = getTheme(props).palette.error.main;
+    return { errorMainColor };
+})`
     font-size: ${FONT_SIZE_TEXT_MEDIUM};
     margin-top: ${MARGIN_X_SMALL};
     align-self: center;
-    color: ${COLOR_ERROR};
+    color: ${props => props.error};
```

Comment:
This error container is the same in all form components, then let's remove this file and use this one:
```src/components/Input/styled/errorText.js```
in order to reuse.